data/strings/strings.txt

```diff
@@ -1846,6 +1846,48 @@
     zh-Hans = 书签
     zh-Hant = 書籤
 
+  [bookmarks_and_tracks]
+    comment = "Bookmarks and Tracks" dialog title
+    tags = android,ios
+    en = Bookmarks and Tracks
+    ar = الإشارات المرجعية والمسارات
+    be = Закладкі і маршруты
+    bg = Отметки и маршрути
+    ca = Marcadors i traces
+    cs = Záložky a stopy
+    da = Bogmærker og ruter
+    de = Lesezeichen und Strecken
+    el = Σελιδοδείκτες και διαδρομές
+    es = Marcadores y pistas
+    et = Järjehoidjad ja rajad
+    eu = Laster-markak eta ibilbideak
+    fa = نشانک ها و مسیر
+    fi = Kirjanmerkit ja reitit
+    fr = Signets et parcours
+    he = סימניות ונתיבים
+    hu = Könyvjelzők és csoportok
+    id = Penanda dan Jalur
+    it = Luoghi preferiti e percorsi
+    ja = ブックマークとトラック
+    ko = 북마크 및 트랙
+    mr = बुकमार्क आणि ट्रॅक
+    nb = Bokmerker og ruter
+    nl = Bladwijzers en tracks
+    pl = Zakładki i trasy
+    pt = Favoritos e percursos
+    pt-BR = Favoritos e percursos
+    ro = Locuri preferate și trasee
+    ru = Закладки и треки
+    sk = Záložky a stopy
+    sv = Bokmärken och rutter
+    sw = Alamisho na Nyimbo
+    th = บุ๊กมาร์กและการติดตาม
+    tr = Yer İmleri ve Kayıtlar
+    uk = Мітки та Маршрути
+    vi = Dấu trang và bản vết
+    zh-Hans = 书签和轨迹
+    zh-Hant = 書籤和軌跡
```

Comment:
Please try to reuse existing terminology instead of introducing new names for bookmarks or tracks. Please check existing translations for `[tracks]` and `[bookmarks]`